vars/dockerExecute.groovy

```diff
@@ -126,3 +124,8 @@ def getContainerDefined(script, dockerImage) {
     def k8sMapping = ConfigurationLoader.generalConfiguration(script)?.kubernetes?.k8sMapping ?: [:]
     return k8sMapping[env.POD_NAME]?.get(dockerImage)
 }
+
+@NonCPS
+boolean inKubernetes(script) {
+    return ConfigurationLoader.generalConfiguration(script)?.kubernetes?.enabled ?: false
```

Comment:
I would prefer an environment variable, because the configuration file should best be infrastructure independent: Pipeline should run on both environments without changes.
What about another environment variable than jaas_owner?

Reply:
I have a different opinion here. Weather to run on K8S or not should be specific to the pipeline. On the same Jenkins, only one project can make use of K8S.